### ISSUE TITLE
A different way to manage subscriptions and play lists

### ISSUE THREAD

**Issue by ViktorKahu** [lamarios/clipious#422]
**Is your feature request related to a problem? Please describe.**

When switching to a different instance, all subscriptions and play lists need to be set up again. For example, the instance that I normally use is currently broken and I need to use another one. All my play lists and subscriptions are empty.

**Describe the solution you'd like**

Subscriptions and play lists should be tied to a user account, rather than an instance.

**Describe alternatives you've considered**

Nil

**Additional context**

All subscriptions and playlists in Libre Tube are kept in the app and you can switch to any piped instance. This approach makes a lot more sense.

**Comment by lamarios** [comment]
Already discussed that many times. This app is an invidious client. Invidious has accounts with subscription and playlists management so I'm going to use that.

I'm not planning to do local subs and playlists at the moment.

**Comment by Gamer750** [comment]
> Already discussed that many times. This app is an invidious client. Invidious has accounts with subscription and playlists management so I'm going to use that.
> 
> I'm not planning to do local subs and playlists at the moment.

Bro it don't be rude. This feature may bring so many people to Foss community . And this app could be alternative to newpipe

**Comment by lamarios** [comment]
How was my message rude? Is saying no to a feature request rude?

I think some people are missing the goal of this app. When I started this project I wanted an invidious client so I built it. Ithought it was cool so I shared it.

I'm not looking for the app to be popular or an alternative to something else. If the features you're looking for are already in another app use that app. There's no need to have many app that do the same thing. People working on those do that for free on their free time so we might has well build things that do not exist yet.

I can't say "Yes my lord" to every feature requests, I need to stay focused on the goals of the app. When I have time to work on the app I look at the issues and work on those that are interesting, really needed or ideally both. Working on this needs to remain enjoyable if not the app will end up abandoned quickly.
